boards/arm/stm32/stm32f429i-disco/Kconfig

```diff
@@ -86,7 +86,7 @@ config STM32F429I_DISCO_RAMMTD_SIZE
 
 config STM32F429IDISCO_USBHOST_STACKSIZE
 	int "USB host waiter stack size"
-	default 1024
```

Comment:
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.